release.py

```diff
@@ -0,0 +1,116 @@
+"""Data format for Release."""
+
+
+class State(object):
+  UNUSED_STATUS = 0
+  PENDING = 1
+  FINISHED = 2
+  FAILED = 3
+  ABANDONED = 4
+
+
+class Release(object):
+  """Class for Release Object."""
+
+  def __init__(self, data=None):
+    """Initializes a release.
+
+    Args:
+      data: dictionary with all the release data (optional).
+    """
+    self._release = data or {}
```

Comment:
so setters checks the data but not the constructor so you might want to cast all the properties.

Reply:
and you might want to validate as well from the constructor

Reply:
if I validate all the properties in the constructor and the setters, should I still cast in the get methods or assume all the types will be correct?